language-support/java/codegen/src/it/daml/Tests/Escape.daml

```diff
@@ -26,6 +27,10 @@ data EscapeRecords int void synchronized = EscapeRecords
 data EscapeVariants abstract new = New new
                                  | Abstract abstract
                                  | VarRecord with abstract : abstract; new : new
+  deriving (Eq, Show)
+
+data FieldVsPackage = FieldVsPackage with tests: ()
```

Comment:
Here the field name `tests` is a lower-case version of the package name `Tests`, so it checks there's no clash between them in the generated java.